### DISCUSSION TITLE
Add CI

### DISCUSSION THREAD

**Discussion by theavege** [URUWorks/TeroSubtitler#343] [Ideas]
Hello.

I like your project.
I want to add [CI](https://github.com/andreaspeters/aprsmap/pull/1) for your project.
What do you think about this?

Thanks.

**Comment by chenlung** [comment]
Hello!

I just wanted to respond (as a moderator) to say thank you for your recent contributions. However, it is for the developer (@URUWorks) to manage, so I'm hoping he'll respond to you soon (and I'm aware that the project hasn't received updates for a few months).
